src/analytics/gtag.js

```diff
@@ -101,7 +101,7 @@ const trackExploredUsageClick = (link) => {
 }
 // tracks internal and external link clicks
 const trackLink = (link) => {
-  if (link.startsWith('http')) {
+  if (typeof link === 'string' && link.startsWith('http')) {
```

Comment:
I've added this type check to prevent an error encountered during the development, thank you!

Reply:
I mistakenly referred to `SearchCardFooter` instead of `SearchCardHeader`. Apologies for the confusion! I've now updated `SearchCardHeader` to ensure the `href` properly passes the formatted string and remove the type check for `link` from `gtag.trackLink`. Thank you David 👍!